pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -270,6 +271,33 @@ func (c *Cluster) startProvisioning(config *provisionConfig) {
 			}
 		}
 	}
+
+	// Deploying test based on the storageclassDeviceSet
+	// Todo: Replace the below logic to deploy osd pods over pvcs
+	for _, storageclassDeviceSet := range c.DesiredStorage.Storage {
+		for i := 0; i < storageclassDeviceSet.Count; i++ {
+			sc := "test-sc"
+			pvc := &v1.PersistentVolumeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "pvc-" + storageclassDeviceSet.Name + fmt.Sprintf("-%v", i),
+					Namespace: "rook-ceph",
+				},
+				Spec: v1.PersistentVolumeClaimSpec{
+					AccessModes: []v1.PersistentVolumeAccessMode{v1.ReadWriteOnce, v1.ReadOnlyMany},
+					Resources: v1.ResourceRequirements{
+						Requests: v1.ResourceList{
```

Comment:
@leseb can you point to an example of setting the minimum memory limits for OSDs?

Reply:
@travisn this? https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/osd/osd.go#L131-L136